includes/wikia/api/DesignSystemApiController.class.php

```diff
@@ -165,4 +166,15 @@ private function addCachingHeaders() {
 			$this->response->setCacheValidity( WikiaResponse::CACHE_VERY_SHORT );
 		}
 	}
+
+	private function isWikiaOrgCommunity() {
```

Comment:
I'm a little confused by this method. Can we not just check `wgIsInWikiaOrgProgram`?

Reply:
I consulted platform on this, they recommended to use domain recognition instead of wg variables :) I'll double check with them though.